src/IFrameNavigator.ts

```diff
@@ -1441,10 +1451,23 @@ export default class IFrameNavigator implements Navigator {
     }
   }
 
-  private navigate(readingPosition: ReadingPosition): void {
+  private async navigate(readingPosition: ReadingPosition): Promise<void> {
     this.hideIframeContents();
     this.showLoadingMessageAfterDelay();
     this.newPosition = readingPosition;
+
+    const baseUrl = this.manifestUrl.href.replace(/[a-z]+.opf/, "");
```

Comment:
I'm not sure if this is too hacky to use to shorten from the full path of the resource URL to the short path for that specific file. It assumes the manifest URL for exploded EPubs will be formatted like www.example.com/{library}/{bookname}/{OEBPS}/{packageFile}.opf (`{}` between the brackets can be anything) or 
www.example.com/{...}/{packageFile}.opf and that the resource files will be in the same directory as the packageFile (i.e., OEBPS).

Reply:
I think we can assume that the resources are in the same directory as the manifest.  However, I think we shouldn't assume anything about the structure of the manifest url.  Like, it could be www.example.com/stuff/morestuff/morestuff/morestuff/package.opf

Reply:
Scratch that -- I think we ... might need to read all of this using an epub `container.xml` since that's guaranteed to tell us what we need
http://idpf.org/epub/301/spec/epub-ocf.html#sec-container-metainf-container.xml

So I think it would be more correct to pass the viewer either 1) a link to the manifest.json (for webpubs) or 2) a link to the container.xml (for epubs)

Reply:
Support for passing container.xml in addition to .ops file has been added